Cargo.toml

```diff
@@ -115,3 +115,6 @@ derive_more = { version = "2.0.1", features = ["display"] }
 paste = "1.0.15"
 strum = "0.27"
 strum_macros = "0.27"
+
+[patch.crates-io]
+cairo-annotations = { git = "https://github.com/software-mansion/cairo-annotations.git", branch = "szymczyk/l2-gas" }
```

Comment:
will remove after new version is released
https://github.com/software-mansion/cairo-annotations/pull/95

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/3645/commits/bba8d464db4d5c9c67c277e036a5f18e49fa0ed0